drivers/display/ft8xx/ft8xx.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2020 Hubert Miś
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/display/ft8xx/ft8xx.h>
+
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
I think these are not needed (e.g. stdint)

Reply:
`stddef.h` is needed to define `NULL` used in this module. `stdint.h` defines types like `uint8_t` and `uint32_t` used in this module. Are these headers includes indirectly and by convention not included in the driver modules?

Reply:
these facilities are already provided when using Zephyr (e.g. stding types are used everywhere now)